### PR TITLE
fixed to_hdf

### PR DESCRIPTION
**Pull request by ctw**

### CODE REVIEW

File: ptsa/data/timeseries.py [hunk 1]
```diff
@@ -86,17 +86,20 @@ def to_hdf(self, filename, mode='w', compression=None,
...
         ----------
-        filename : str
-            Full path to the HDF5 file
-        mode : str
-            File mode to use. See the :mod:`h5py` documentation for details.
+        filename : str or :mod:`h5py.h5f.FileID` instance
+            Path to or identifier for the HDF5 file. See the
+            :mod:`h5py.File` documentation.
+        mode : str, optional
+            File mode to use. See the :mod:`h5py.File` documentation
```
Comment (mivade) [inline ptsa/data/timeseries.py]: This should read `:class:` rather than `:mod:`

File: ptsa/data/timeseries.py [hunk 2]
```diff
@@ -86,17 +86,20 @@ def to_hdf(self, filename, mode='w', compression=None,
...
+            for details.
             Default: ``'w'``
         compression : str or None
-            Compression to use with arrays (see :mod:`h5py` documentation for
-            valid choices).
+            Compression to use with arrays (see
+            :mod:`h5py.Group.create_dataset` documentation for valid
```
Comment (mivade) [inline ptsa/data/timeseries.py]: `:function:` or `:meth:`

File: ptsa/data/timeseries.py [hunk 3]
```diff
@@ -86,17 +86,20 @@ def to_hdf(self, filename, mode='w', compression=None,
...
+            choices).
         compression_opts : int or None
             Compression options, generally a number specifying compression level
-            (see :mod:`h5py` documentation for details).
+            (see :mod:`h5py.Group.create_dataset` documentation for details).
```
Comment (mivade) [inline ptsa/data/timeseries.py]: See above.

Comment (ctw) [reply ptsa/data/timeseries.py]: Thanks -- fixed now!

**Pull request merged**